var/spack/repos/builtin/packages/bueno/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Bueno(PythonPackage):
```

Comment:
FWIW.  We generally prefer python package names to start with `Py`.  And a package, say `PyBueno`, would reside in the `var/spack/repos/builtin/packages/py-bueno/` spack subdirectory.

Reply:
See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name for the official guidance on this.

Reply:
I was considering the `py-bueno` name, but from my point of view the way it is used is via the `bueno` command line utility. It has little use as just a Python package. @samuelkgutierrez how do you think about this?

Reply:
> See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name for the official guidance on this.

I didn't realize (remember?) there was an official guide on this.  Thanks Adam!